modules/ocf_ceph/manifests/mon.pp

```diff
@@ -0,0 +1,92 @@
+# Ceph monitor + manager + metadata server configuration
+# For now, each monitor (mon) is also a manager (mgr) and a metadata server (mds)
+
+class ocf_ceph::mon {
+  require ocf_ceph
+  require ocf_ceph::admin
+
+  $mon_key = lookup('ceph::mon_key')
+  $admin_key = lookup('ceph::admin_key')
+  $bootstrap_osd_key = lookup('ceph::bootstrap_osd_key')
+
+  file { [
+      '/var/lib/ceph/mon',
+      '/var/lib/ceph/mgr',
+      '/var/lib/ceph/mds',
+      "/var/lib/ceph/mon/ceph-${::hostname}",
+      "/var/lib/ceph/mgr/ceph-${::hostname}",
+      "/var/lib/ceph/mds/ceph-${::hostname}",
+    ]:
+      ensure => directory,
+      owner  => 'ceph',
+      group  => 'ceph',
+  }
+
+  # Allow Ceph monitor communications
+  ocf::firewall::firewall46 {
+    '101 allow ceph monitors':
+      opts => {
+        chain  => 'PUPPET-INPUT',
+        proto  => 'tcp',
+        dport  => 3300,
+        action => 'accept',
+      };
```

Comment:
can we permit access from mons only?

Reply:
This needs to be access from any ceph-using machine. We could have a puppetdb query for that I suppose

Reply:
ah, maybe not worth it in that case